licenses/migrations/0001_initial.py

```diff
@@ -47,6 +47,7 @@ class Migration(migrations.Migration):
                 (
                     "translation_last_update",
                     models.DateTimeField(
+                        blank=True,
```

Comment:
Rather than manually editing the migration files, it would be less error-prone to let Django auto-generate the migrations. If the number of migrations is a concern, perhaps it would be reasonable to [squash migrations](https://docs.djangoproject.com/en/3.2/topics/migrations/#migration-squashing) every so often.